diff -ru milkytracker-0.90.85.orig/src/compression/DecompressorGZIP.cpp milkytracker-0.90.85/src/compression/DecompressorGZIP.cpp
--- milkytracker-0.90.85.orig/src/compression/DecompressorGZIP.cpp	2009-04-17 23:34:16.000000000 +0300
+++ milkytracker-0.90.85/src/compression/DecompressorGZIP.cpp	2012-08-03 21:07:31.440952993 +0300
@@ -57,11 +57,11 @@
 	
 bool DecompressorGZIP::decompress(const PPSystemString& outFileName, Hints hint)
 {
-    gzFile *gz_input_file = NULL;
+	gzFile gz_input_file = NULL;
 	int len = 0;
 	pp_uint8 *buf;
 	
-	if ((gz_input_file = (void **)gzopen (fileName.getStrBuffer(), "r")) == NULL)
+	if ((gz_input_file = gzopen (fileName.getStrBuffer(), "r")) == NULL)
 		return false;
 	
 	if ((buf = new pp_uint8[0x10000]) == NULL)
