diff -ur milkytracker-0.90.85-vanilla/configure.in milkytracker-0.90.85-syslibs/configure.in
--- milkytracker-0.90.85-vanilla/configure.in	2009-09-17 22:35:47.000000000 +0300
+++ milkytracker-0.90.85-syslibs/configure.in	2011-06-07 21:38:49.661911384 +0300
@@ -116,7 +116,6 @@
 AC_CONFIG_FILES([Makefile
                  src/Makefile
                  src/compression/Makefile
-                 src/compression/zziplib/generic/Makefile
                  src/fx/Makefile
                  src/midi/Makefile
                  src/milkyplay/Makefile
diff -ur milkytracker-0.90.85-vanilla/src/compression/Makefile.am milkytracker-0.90.85-syslibs/src/compression/Makefile.am
--- milkytracker-0.90.85-vanilla/src/compression/Makefile.am	2009-09-17 22:35:47.000000000 +0300
+++ milkytracker-0.90.85-syslibs/src/compression/Makefile.am	2011-06-07 21:51:05.203584868 +0300
@@ -1,4 +1,4 @@
-SUBDIRS = zziplib/generic
+# SUBDIRS = zziplib/generic
 noinst_LIBRARIES = libcompression.a
 
 libcompression_a_SOURCES =  PP20.cpp ZipExtractor.cpp zziplib/MyIO.cpp \
diff -ur milkytracker-0.90.85-vanilla/src/compression/ZipExtractor.cpp milkytracker-0.90.85-syslibs/src/compression/ZipExtractor.cpp
--- milkytracker-0.90.85-vanilla/src/compression/ZipExtractor.cpp	2009-04-17 23:34:16.000000000 +0300
+++ milkytracker-0.90.85-syslibs/src/compression/ZipExtractor.cpp	2011-06-07 22:27:43.661844462 +0300
@@ -35,7 +35,8 @@
 #include "XModule.h"
 #include "XMFile.h"
 #include "MyIO.h"
-#include "zzip_lib.h"
+/*#include "zzip_lib.h"*/
+#include <zzip/lib.h>
 
 ZipExtractor::ZipExtractor(const PPSystemString& archivePath) :
 	archivePath(archivePath)
diff -ur milkytracker-0.90.85-vanilla/src/compression/zziplib/MyIO.h milkytracker-0.90.85-syslibs/src/compression/zziplib/MyIO.h
--- milkytracker-0.90.85-vanilla/src/compression/zziplib/MyIO.h	2009-01-05 22:27:42.000000000 +0200
+++ milkytracker-0.90.85-syslibs/src/compression/zziplib/MyIO.h	2011-06-07 22:08:57.127295361 +0300
@@ -9,7 +9,9 @@
 #ifndef __MYIO_H__
 #define __MYIO_H__
 
-#include "zzip_plugin.h"
+#include <zzip/plugin.h>
+
+#define __zzipfd size_t
 
 #ifdef __cplusplus
 extern "C" {
diff -ur milkytracker-0.90.85-vanilla/src/tracker/Makefile.am milkytracker-0.90.85-syslibs/src/tracker/Makefile.am
--- milkytracker-0.90.85-vanilla/src/tracker/Makefile.am	2009-09-17 22:35:47.000000000 +0300
+++ milkytracker-0.90.85-syslibs/src/tracker/Makefile.am	2011-06-07 22:48:54.997077361 +0300
@@ -74,9 +74,9 @@
 $(top_builddir)/src/ppui/libppui.a \
 $(top_builddir)/src/ppui/osinterface/libosinterface.a \
 $(top_builddir)/src/fx/libfx.a $(compressionLib) $(RTMIDI) $(SDL_LIBS) \
-$(top_builddir)/src/compression/zziplib/generic/libzzip.a \
-$(Z_LIBS)
-#$(ZZIP_LIBS) read below
+$(Z_LIBS) \
+$(ZZIP_LIBS)
+# read below
 
 # NOTE: The MilkyTracker source contains a MODIFIED version of the zzip
 # library, which is statically linked to the executable.  Simply changing the
